### PR TITLE
DOC Note that module is not compatible with CMS5

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # HTML5 support for SilverStripe
 
+__NOTE__: This module not longer commercially supported in Silverstripe CMS 5 and it does not provide CMS5-compatible versions.
```
Comment (michalkleiner) [inline README.md]:
```suggestion
__NOTE__: This module is no longer commercially supported in Silverstripe CMS 5 and it does not provide a CMS5-compatible version.
```

Comment (sabina-talipova) [reply README.md]: Done

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Remove supported module badge.

**Comment by sabina-talipova** [comment]
Updated

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**